internal/algod/linux/linux.go

```diff
@@ -47,7 +63,12 @@ func InstallRequirements() system.CmdsList {
 
 // Install installs Algorand development tools or node software depending on the package manager.
 func Install() error {
+	if hasConflictingUser() {
+		log.Fatal("Your system has a user called \"algorand\". The algorand node requires the \"algorand\" username for internal usage. Rename or remove the algorand user to continue.")
```

Comment:
Since Install returns an error we don't want to panic/exit here. 

```suggestion
		return fmt.Errorf("Your system has a user called \"algorand\". The algorand node requires the \"algorand\" username for internal usage. Rename or remove the algorand user to continue.")
```